models/node.go

```diff
@@ -0,0 +1,5 @@
+package models
+
+const (
+	NodeSystemNamespace string = "system"
```

Comment:
Worth adding a comment to indicate some more context on what the "system" means in this context. I saw above you added a template variable called `adminNamespace`.

Reply:
yea, I named it admin in the CLI because I though that was more clear, but I didn't want to name it "admin" in the internals.  I will add some comments

Reply:
[2ab1cef](https://github.com/synadia-io/nex/pull/469/commits/2ab1cefc4bfbf22af78cf44813405821418d4101)